GAE/war/auth0/auth0.html

```diff
@@ -0,0 +1,112 @@
+<title>Akvo Flow - Auth</title>
+<link rel="shortcut icon" href="/favicon.ico" />
+<!--[if lt IE 9]>
+      <script src="https://html5shiv.googlecode.com/svn/trunk/html5.js"></script>
+    <![endif]-->
+
+<style>
+body, html {
+	margin: 0;
+	padding: 0;
+	font-family: 'Helvetica Neue', helvetica, Arial, sans-serif;
+
+}
+.msgWrap {
+	position: relative;
+	width: 100%;
+	height: 100%;
+	background: #fffff;
+}
+.msgContainer {
+	position: absolute;
+	max-height: calc(100vh - 40px);
+	width: 25%;
+    min-height: auto;
+    height: auto;
+    top: 50%;
+    left: 50%; 
+    padding:20px 0;
+    -webkit-transform: translate(-50%, -50%);
+    -moz-transform: translate(-50%, -50%); 
+    -ms-transform: translate(-50%, -50%); 
+    -o-transform: translate(-50%, -50%); 
+    transform: translate(-50%, -50%); 
+    box-shadow: 0 0 20px rgba(32,32,36,0.2);
+    }
+h1 {
+	padding: 0 10px;
+	text-align: center;
+	font-size: 1.2em;
+}
+p {
+	text-align: center;	
+	font-size: 1.2em;
+
+}
+p:after {
+	content: "";
+	width: 50%;
+	border-bottom: 1px solid rgba(100,100,100,0.5);
+	display: block;
+	margin: 3em auto;
+	padding: 0 5px;
+}
+img {
+	position: relative;
+	display: block;
+	width:25%;
+	margin: 0 auto 1em auto;
+	max-width: 280px;
+	min-width:150px;
+}
+#urlLoc {
+	font-weight: normal;
+}
+</style>
+</head>
+<body>
+<div class="msgWrap"> 
+  	<div class="msgContainer">
+  		<img src="https://akvo-public.storage.googleapis.com/AkvoLogo.svg" title="AKVO" alt="Akvo.org logo" />
+		<h1><span id="errorTitle"></span></h1>
+		<p><span id="errorCode"></span></p>
+	</div>
+</div>
+<script type="text/javascript">
+
+  var messages = {
+      "es" :
+      { "ERROR_TITLE":"Existe un problema verificando su identidad:",
+	"EMAIL_VERIFIED_ERROR": "Necesita verificar su dirección de email, por favor revise su buzón de correo y haga click en el enlace de verificación"},
+      "en" :
+      { "ERROR_TITLE":"There was an issue verifying your identity",
+	"EMAIL_VERIFIED_ERROR": "You need to verify your email address, please check your inbox email and click the verification link."},
+      "fr" :
+      { "ERROR_TITLE":"Un problème est survenu lors de la vérification de votre identité",
+	"EMAIL_VERIFIED_ERROR": "Vous devez vérifier votre adresse e-mail. Vérifiez votre boîte de courrier electronique et cliquez sur le lien de vérification"}
+  };
+
+  const langs = new Set(['en', 'es', 'fr']);
+
+  function errorCode(data) {
+      var urlParams = new URLSearchParams(window.location.search);
+      return data[urlParams.get('errorCode')] || "Please contact with support@akvo.org about this problem: "+urlParams.get('errorCode');
+  }
+
+  function lang() {
+      var l = navigator.language.substring(0, 2);
+      if (langs.has(l)) {
+	  return l;
+      } else {
+	  return 'en';
+      }
+  }
+  document.getElementById("errorTitle").innerHTML = messages[lang()]["ERROR_TITLE"];
+  document.getElementById("errorCode").innerHTML = errorCode(messages[lang()]);
```

Comment:
Maybe handle better the case where the error code is null/empty string, as the user is going to see "Please contact with support@akvo.org about this problem: null"